### PR TITLE
Add feature test submit post

### PR DESCRIPTION
**Pull request by danieljonesdmj**

### CODE REVIEW

File: spec/features/user_can_submit_posts_spec.rb [hunk 1]
```diff
@@ -1,14 +1,30 @@
...
+#     visit "/posts"
+#     click_link "New post"
+#     fill_in "Message", with: "Hello, coppers!\bDid you miss me?"
+#     click_button "Submit"
+#     # expect(page).to have_content("Hello, world!\bDid you miss me?")
+#     expect(page).to have_css('p', text: 'Hello, coppers!')
+#     expect(page).to have_css('p', text: 'Did you miss me?')
+#     expect(page).to have_content('Posted less than a minute ago')
+#   end
+# end
```
Comment (heuperman) [inline spec/features/user_can_submit_posts_spec.rb]: Why is this test commented out?

File: spec/features/user_can_submit_posts_spec.rb [hunk 2]
```diff
@@ -1,14 +1,30 @@
...
-    expect(page).to have_css('p', text: 'Hello, coppers!')
-    expect(page).to have_css('p', text: 'Did you miss me?')
-    expect(page).to have_content('Posted less than a minute ago')
+  sign_up_test
+  click_link 'New post'
+  expect(page).to have_content("Message")
+
+  fill_in "Message", with: "Hello, coppers!\bDid you miss me?"
+  click_button "Submit"
+  # expect(page).to have_content("Hello, world!\bDid you miss me?")
```
Comment (heuperman) [inline spec/features/user_can_submit_posts_spec.rb]: And this line?

File: spec/helpers/comments_helper_spec.rb [hunk 3]
```diff
@@ -11,5 +11,5 @@
 #   end
 # end
 RSpec.describe CommentsHelper, type: :helper do
-  pending "add some examples to (or delete) #{__FILE__}"
+  # pending "add some examples to (or delete) #{__FILE__}"
```
Comment (heuperman) [inline spec/helpers/comments_helper_spec.rb]: Just get rid of this?

File: spec/helpers/users_helper_spec.rb [hunk 4]
```diff
@@ -3,13 +3,17 @@
...
+  fill_in 'user_username', with: 'test'
+  fill_in 'user_email', with: email
+  fill_in 'user_password', with: 'hello1'
+  fill_in 'user_password_confirmation', with: 'hello1'
+  click_on 'Sign up'
+end
+
 RSpec.describe UsersHelper, type: :helper do
-  pending "add some examples to (or delete) #{__FILE__}"
+  # pending "add some examples to (or delete) #{__FILE__}"
```
Comment (heuperman) [inline spec/helpers/users_helper_spec.rb]: Remove this one as well

File: spec/features/user_can_submit_posts_spec.rb (hunk 1 above)
Comment (bibicollins) [reply spec/features/user_can_submit_posts_spec.rb]: We changed the test, but forgot to get rid of it.. All done!

**Pull request merged**